### DISCUSSION TITLE
offer/experience/2023summary

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#178] [Announcements]
# offer/experience/2023summary

时间不在于你拥有多少，而在于你怎样使用

https://sugarat.top/offer/experience/2023summary.html

**Comment by lineo1227** [comment]
我在摸🐟

**Comment by immelon0097** [comment]
你好，你的周刊和博客内容很好。但评论系统配置似乎存在一些小问题，表现为二级目录下所有的 giscus 互动似乎是混合在一起的。

具体来说就是 weekly 下所有周刊的评论和互动都是同步的，这是预期之外的事情。

**Reply by ATQQ** [reply]
感谢反馈，我下来看一下是什么问题导致的

**Reply by ATQQ** [reply]
看了一下，评论本身没有问题。

问题在展示这块，在切换路由的时候没有更新，表现就是上面这种效果。
